src/lib/common/macroSubstitute.cpp

```diff
@@ -121,8 +121,16 @@ static void attributeValue(std::string* valueP, const std::vector<ContextAttribu
 *
 */
 #define CHUNK_SIZE 1024
-void macroSubstitute(std::string* to, const std::string& from, const ContextElement& ce)
+bool macroSubstitute(std::string* to, const std::string& from, const ContextElement& ce)
 {
+  // Initial size check: is the string to convert too big?
```

Comment:
Although I understand it is not really a big problem mark as overpass the case in which the template is >8MB but the actual payload after substitution <8MB, is this really needed? I mean, if this check is removed, does the rest of the logic breaks?